atomics/T1053.006/T1053.006.yaml

```diff
@@ -0,0 +1,54 @@
+attack_technique: T1053.006
+display_name: 'Scheduled Task/Job: Systemd Timers'
+
+atomic_tests:
+- name: Create Systemd Service and Timer
+  auto_generated_guid: 42d33451-0a3c-4292-b753-9dbdcd67a0d3
```

Comment:
Please remove the `auto_generated_guid` line so that the build system can generate a unique value when your PR is merged. This is the best way to avoid conflicts.